### DISCUSSION TITLE
Add 'Show top 1000 Table records'

### DISCUSSION THREAD

**Discussion by pbower** [mtxr/vscode-sqltools#817] [Ideas]
Hi ,

I am a big fan of SQL Tools. Thanks so much for building such an awesome exploration / query building plugin for VSCode.

I'm wondering if it would be possible to request a feature similar to 'right-click: Show Table Records' that currently appears, and add a 'Show Top 1000 Table Records' similar to 'SELECT * FROM TABLE LIMIT 1000;' as featured in most query software?

Thanks
Pete

**Comment by gjsjohnmurray** [comment]
The `previewLimit` property of a connection definition already controls how many records are fetched by `Show Table Records`.
